kustomize/commands/version/version.go

```diff
@@ -62,12 +63,24 @@ func (o *Options) Validate(_ []string) error {
 }
 
 func (o *Options) Run() error {
+	bi, ok := debug.ReadBuildInfo()
+	if !ok {
+		return fmt.Errorf("unable to read build information")
+	}
+
+	var version string
+	for _, dep := range bi.Deps {
+		if dep.Path == "sigs.k8s.io/kustomize/kustomize" {
+			version = dep.Version
+			break
+		}
+	}
```

Comment:
This change mirrors very closely the code in `provenance.go`: https://github.com/kubernetes-sigs/kustomize/blob/bfb00ecb2747dc711abfc27d9cf788ca1d7c637b/api/provenance/provenance.go#L53-L64

I think this code should be incorporated in that place, instead of added here.

Reply:
@stormqueen1990 Thank you for your review!! I will update the function.